source/initial_conditions/adiabatic_boundary.cc

```diff
@@ -135,10 +137,13 @@ namespace aspect
       {
         prm.enter_subsection("Adiabatic boundary");
         {
+          prm.declare_entry ("Data directory", "$ASPECT_SOURCE_DIR/tests/adiabatic_boundary/",
```

Comment:
I would prefer if the default value could be set to "$ASPECT_SOURCE_DIR/data/initial-conditions/adiabatic-boundary/", and you would move the test data file there as well. This is how we handle it for all the other tests that need data files, and then all data is in one subfolder (aspect/data).